src/leap/common/events/README.rst

```diff
@@ -22,32 +25,29 @@ How to use it
 To start the events server:
 
 >>> from leap.common.events import server
->>> server.ensure_server(port=8090)
+>>> server.ensure_server(sig_port=9000, pub_port=9001)
 
 To register a callback to be called when a given signal is raised:
 
 >>> from leap.common.events import (
 >>>     register,
->>>     events_pb2 as proto,
+>>>     signals,
 >>> )
 >>>
->>> def mycallback(sigreq):
->>>     print str(sigreq)
+>>> def mycallback(sig, content):
+>>>     print str((sig, content))
 >>>
->>> events.register(signal=proto.CLIENT_UID, callback=mycallback)
+>>> events.register(signal=signals.CLIENT_UID, callback=mycallback)
 
 To signal an event:
 
 >>> from leap.common.events import (
 >>>     signal,
->>>     events_pb2 as proto,
+>>>     signals,
 >>> )
->>> signal(proto.CLIENT_UID)
+>>> signal(signals.CLIENT_UID)
 
 Adding events
 -------------
 
-* Add the new event under enum ``Event`` in ``events.proto`` 
-* Compile the new protocolbuffers file::
-
-  make
+To add a new event, just add it to ``signals.py``.
```

Comment:
in the past, we've avoided the naming of "signals" so it cannot be confused with Qt signals.
maybe events.catalog or something like that?